commcare_connect/opportunity/models.py

```diff
@@ -166,6 +174,7 @@ class VisitValidationStatus(models.TextChoices):
     pending = "pending", gettext("Pending")
     approved = "approved", gettext("Approved")
     rejected = "rejected", gettext("Rejected")
+    extra = "extra", gettext("Extra")
```

Comment:
What about calling this 'Over Limit' or something? 'Extra' seems a bit vague to me.

Reply:
Yeah, that makes more sense. Renamed. https://github.com/dimagi/commcare-connect/pull/146/commits/d23c83ccfe5a61d050f95a6a440ec974836bb4b2